src/Symfony/Bundle/WebProfilerBundle/Tests/Resources/IconTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Tests\Resources;
+
+use Symfony\Bundle\WebProfilerBundle\Tests\TestCase;
```

Comment:
use the PHPUnit class directly instead IMO. this base testcase is useless as it is empty